zerver/webhooks/uptimerobot/doc.md

```diff
@@ -0,0 +1,36 @@
+Receive Zulip notifications for your Uptime Robot monitors!
+
+1. {!create-stream.md!}
+
+1. {!create-bot-construct-url-indented.md!}
+
+    Make sure to specify the topic in the URL above. Otherwise, the
+    default topic `General` will be used.
```

Comment:
You can delete this sentence (see changes below)

Reply:
Yeah deleted to keep in line with other changes.